src/pages/students/students.scheme.ts

```diff
@@ -0,0 +1,42 @@
+import * as yup from "yup";
+
+export const studentsScheme = yup
+  .object({
+    name: yup.string().required("Debe ingresar el nombre"),
+    lastname: yup.string().required("Debe ingresar el apellido"),
+    nroTelefono: yup
+      .string()
+      .min(8, "Debe tener al menos 8 caracteres")
+      .max(15, "No puede superar los 15 caracteres")
+      .matches(/^[+0-9\s-]+$/, "Solo se permiten números"),
+    fechaNacimiento: yup
+      .date()
+      .nullable()
+      .required("La fecha de nacimiento es obligatoria")
```

Comment:
Como mencione antes, estas dos validaciones juntas no tienen sentido, porque o es requerido o puede ser nulleable
```suggestion
      .required("La fecha de nacimiento es obligatoria")
```